cli/UploadSourcemaps.ts

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import { Command, Option } from 'commander';
+import FormData from 'form-data';
+import fs from 'fs';
+import path from 'path';
+
+export const uploadSourcemapsCommand: Command = new Command();
```

Comment:
**suggestion:** We can remove the type annotation, since it's already interfered.

```suggestion
export const uploadSourcemapsCommand = new Command();
```